roles/lnls-ans-role-epics7/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 epics7_base_version: 7.0.3
 epics7_base_arch: linux-x86_64
-epics7_base_url: "https://epics.anl.gov/download/base"
+# epics7_base_url: "https://epics.anl.gov/download/base"
```

Comment:
I think we should use the previous sensible defaults for this. If wanted we can set this variable elsewhere and ansible will override it automatically